python_frank_energie/frank_energie.py

```
@@ -1586,15 +1704,19 @@ def _validate_start_date_format(self, start_date: str | date) -> None:
+            )
 
         if len(start_date) == 10:  # volledige datum
             try:
                 date_obj = datetime.strptime(start_date, "%Y-%m-%d").date()
                 if date_obj > datetime.now(timezone.utc).date():
                     raise ValueError("De 'start_date' mag niet in de toekomst liggen.")
             except ValueError as e:
-                raise ValueError("De 'start_date' heeft geen geldig datumformaat: %s" % e)
+                raise ValueError(
+                    "De 'start_date' heeft geen geldig datumformaat: %s" % e
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)